java-manta-client-unshaded/src/test/java/com/joyent/manta/exception/MantaErrorCodeTest.java

```diff
@@ -14,14 +14,22 @@
  * Tests the behavior of the {@link MantaErrorCode} enumeration.
  *
  * @author <a href="https://github.com/dekobon">Elijah Zupancic</a>
+ * @author <a href="https://github.com/nairashwin952013">Ashwin A Nair</a>
  */
 @Test
 public class MantaErrorCodeTest {
     public void valueOfCodeCanFindByMatchingCode() {
-        final MantaErrorCode expected = MantaErrorCode.INVALID_LIMIT_ERROR;
-        final MantaErrorCode actual = MantaErrorCode.valueOfCode("InvalidLimit");
+        final MantaErrorCode expectedLimitError = MantaErrorCode.INVALID_LIMIT_ERROR;
```

Comment:
I would just use the raw enum because that is simpler.

Reply:
Imported ```raw enums``` **statically** that were used in the added unit tests and the existing ones.